src/main/java/io/reactivex/Single.java

```diff
@@ -2017,6 +2021,8 @@
     /**
      * Delays the emission of the success signal from the current Single by the specified amount.
      * An error signal will not be delayed.
+     * <p>
+     * <img width="640" height="457" src="https://raw.githubusercontent.com/UMFsimke/RxJava/Images/Images/DelayScheduler.png" alt="">
```

Comment:
It would be great if you indicated errors are not delayed with this overload.

Reply:
Please also color the bottom arrows as orange to indicate they get executed on the provided scheduler.